examples/worker-pool.php

```diff
@@ -2,14 +2,26 @@
 <?php
 require dirname(__DIR__).'/vendor/autoload.php';
 
+
```

Comment:
Please remove the unnecessary white space change. Maybe add a newline before the `require`.

Reply:
Fixed.